platform/platform-app/pom.xml

```diff
@@ -348,11 +350,11 @@
         <jackson2-version>2.8.8</jackson2-version>
```

Comment:
It is used via property replacement in the cxf features though. Should I just replace the `${geronimo-jms-spec-version}` at https://github.com/codice/ddf/blob/master/platform/platform-app/pom.xml#L387 with `1.1.1` (what geronimo-jms-spec-version is set to)?